test/test-platform-output.c

```diff
@@ -171,5 +171,6 @@ TEST_IMPL(platform_output) {
   printf("  version: %s\n", uname.version);
   printf("  machine: %s\n", uname.machine);
 
+  uv_os_free_passwd(&pwd);
```

Comment:
For consistency with the other free operations in this test, can you move this up to line 158.